docs/.vuepress/theme/styles/fonts.styl

```diff
@@ -18,3 +18,8 @@
     font-family: 'Amazon Ember';
     src: url('../fonts/AmazonEmber_Rg.ttf');
     font-weight: 400;
+
+@font-face
+    font-family: 'Amazon Ember';
+    src: url('../fonts/AmazonEmber_Md.ttf');
+    font-weight: 600
```

Comment:
You're missing a semicolon and new line ending here.

Reply:
good catch. will update